x-pack/plugins/search_indices/public/components/shared/code_sample.tsx

```diff
@@ -40,10 +49,14 @@ export const CodeSample = ({ id, title, language, code, onCodeCopyClick }: CodeS
 
   return (
     <EuiFlexItem id={id}>
-      <EuiText size="s">
-        <strong>{title}</strong>
+      <EuiTitle size="xs">
+        <h3>{title}</h3>
+      </EuiTitle>
+      <EuiSpacer size="xs" />
+      <EuiText size="s" grow={false}>
+        <p>{description}</p>
       </EuiText>
```

Comment:
since `description` is optional we should check that its set before rendering this group

Reply:
yep good spot. Thanks